src/app/menu/menu.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="col-md-12 text-center"><h2 class="stage-header">{{language?.menu.heading}}</h2></div>
+<div class="col-md-5 pull-right"><h2 class="stage-header">{{language?.menu.heading}}</h2></div>
```

Comment:
Wasn't this change made in your previous PR?

Reply:
Yes.I will just drop the previous commit.